packages/hub/.env.test

```diff
@@ -1,2 +1,2 @@
 HUB_ADDRESS=0x7892ea3A79ffB40388d6769Ef58642b3e4412Fbb
-HUB_PRIVATE_KEY=0xA39E13320C7752AEA43CE032CF40EF6B10BDC7753AA8B98A07F9477F30C8AC8F;
\ No newline at end of file
+HUB_PRIVATE_KEY=0xA39E13320C7752AEA43CE032CF40EF6B10BDC7753AA8B98A07F9477F30C8AC8F;
```

Comment:
Why did the private key need to get updated?

Reply:
What does this even do?

Reply:
The private key was not updated. A newline was added to the end of the file. It would be nice to enforce newlines at the ends of files if that's easy to add.

`HUB_PRIVATE_KEY` env var is not wired up at the moment and can be deleted. I believe the intention of the env var was to store the key used for signing both states and transactions.

Reply:
Removed `.env.test` for now and filed https://github.com/statechannels/monorepo/issues/677.